charts/drax/values.yaml

```diff
@@ -480,6 +480,16 @@ grafana:
   adminUser: accelleran
   adminPassword: accelleran
 
+  grafana.ini:
+    auth.basic:
+      enabled: false
```

Comment:
Does this disable authentication entirely? IIRC, we need the log-in in order to edit or create new dashboards, which I believe the integration team does frequently.

Reply:
We can still login and then start editing/adding dashboards. It's just not enforced anymore (as the anonymous access is there).

Reply:
Let me make myself clearer:
Someone with anonymous access can view (but not edit etc). This will be the default as no login is enforced.
Everyone can sign in (top right button) and then start editing/adding dashboards.

Reply:
Perfect!

Reply:
I've deployed it on the `drax-dev` machine. You can check it out over there :smile: